py/README

```diff
@@ -14,20 +12,20 @@ The `selenium` package is used automate web browser interaction from Python.
 +-----------+-----------------------------------------------------------------------------------+
 | **Docs**: | `selenium package API <http://selenium.googlecode.com/git/docs/api/py/api.html>`_ |
 +-----------+-----------------------------------------------------------------------------------+
-| **Dev**:  | https://github.com/SeleniumHQ/selenium                                               |
+| **Dev**:  | https://github.com/SeleniumHQ/selenium                                            |
 +-----------+-----------------------------------------------------------------------------------+
 | **PyPI**: | https://pypi.python.org/pypi/selenium                                             |
 +-----------+-----------------------------------------------------------------------------------+
 | **IRC**:  | **#selenium** channel on freenode                                                 |
 +-----------+-----------------------------------------------------------------------------------+
 
-Several browsers/drivers are supported (Firefox, Chrome, Internet Explorer, PhantomJS), as well as the Remote protocol.
+Several browsers/drivers are supported (Firefox, Chrome, Internet Explorer, PhantomJS, etc), as well as the Remote protocol.
 
 Supported Python Versions
 =========================
 
-* Python 2.6, 2.7
-* Python 3.2, 3.3
+* Python 2.6+
+* Python 3.2+
```

Comment:
I think "3.2, 3.3, 3.4" is more accurate.

(Has it already been tested with 3.5? Will it automagically be with Python 3.6 once that is released?)